src/services/datahub-client.doubles.ts

```diff
@@ -115,12 +120,22 @@ export class HardcodedDatahubClient implements DatahubClient {
   }: {
     domainName: string;
     dateFrom: Date;
-  }): Promise<TrafficSource[]> {
+  }): Promise<TrafficSourceResult> {
     console.log('getTrafficSourcesByPeriod', { domainName, dateFrom });
     await timeout(1000);
-    return fakeTrafficSourcesData.map((x) => ({
+    const trafficSources = fakeTrafficSourcesData.map((x) => ({
       sourceName: x.sourceName,
       quantity: x.quantity,
     }));
+
+    return {
+      success: true,
+      value: trafficSources,
+    };
+
+    //return {
```

Comment:
Is this ok? (Commented code)

Reply:
yes, we comment the other response options. It's more easy for tests pruposes